troubleshooting/index.md

```diff
@@ -10,6 +10,26 @@ type: markdown
 
 Oh no! You have some problems? Let us help.
 
+## No traces appearing in Zipkin when running Istio locally on Mac
+
+Istio is installed and everything seems to be working except there are no traces showing up in Zipkin when there
+should be.
+
+This may be caused by a known [Docker issue](https://github.com/docker/for-mac/issues/1260) where the time inside
+containers may skew significantly from the time on the host machine. If this is the case, 
+when you select a very long date range in Zipin you will see the traces appearing as much as several days too early.
+
+You can also confirm this problem by comparing the date inside a docker container to outside:
+
+```bash
+$ docker run --entrypoint date gcr.io/istio-testing/ubuntu-16-04-slave:latest
+Sun Jun 11 11:44:18 UTC 2017
+$ date -u
+Thu Jun 15 02:25:42 UTC 2017
+```
+
+To fix the problem, you'll need to shutdown and then restart Docker before reinstalling Istio.
```

Comment:
If you want to drop a CLI solution in here -

`osascript -e 'quit app "Docker"' && open -a Docker`